projects/ng-components-docs/src/app/components/example-page/api-list/api-list.component.ts

```diff
@@ -204,7 +204,7 @@ export class ApiListComponent implements OnInit {
             .map(property => {
                 return {
                     ...property,
-                    description: ApiListComponent.parseComment(outputs[property.name].comment),
+                    description: ApiListComponent.parseComment(outputs[property.name] && outputs[property.name].comment),
```

Comment:
not having a check will make the API documentation fail when you use an optional Output